src/devices/src/virtio/vsock/unix/muxer.rs

```diff
@@ -463,6 +466,7 @@ impl VsockMuxer {
         //   termination.
         self.sweep_killq();
 
+        METRICS.vsock.conns_added.inc();
```

Comment:
Looks like this way we are incrementing `cons_added` even if we are returning an error (in the following `if`). Is this the desired behavior or can we do this `inc()` when we are sure the connection is added?

Reply:
Done